lte/gateway/c/session_manager/LocalSessionManagerHandler.cpp

```diff
@@ -77,8 +77,12 @@ void LocalSessionManagerHandlerImpl::ReportRuleStats(
     MLOG(MINFO) << "Pipelined has been restarted, attempting to sync flows,"
                 << " old epoch = " << current_epoch_
                 << ", new epoch = " << reported_epoch_;
+
     enforcer_->get_event_base().runInEventBaseThread(
-        [this, epoch = reported_epoch_]() { call_setup_pipelined(epoch); });
+        [this, epoch = reported_epoch_,
+         update_rule_versions = request_cpy.update_rule_versions()]() {
+          call_setup_pipelined(epoch, update_rule_versions);
```

Comment:
why not call deactivate flows for all the rules?

Reply:
2 things:
 - when we do this we lose some flow state, basically any traffic since last report will not get reported(because we bump the version and remove old flow)
 - extra processing to remove all / add all rules back (which happen swhen you increment version)